lib/git_reflow/git_server/git_hub.rb

```diff
@@ -86,9 +86,9 @@ def connection
       def authenticate(options = {silent: false})
         if connection and self.class.oauth_token.length > 0
           unless options[:silent]
-            puts "Your GitHub account was already setup with: "
-            puts "\tUser Name: #{self.class.user}"
-            puts "\tEndpoint: #{self.class.api_endpoint}"
+            GitReflow.say "Your GitHub account was already setup with: "
+            GitReflow.say "\tUser Name: #{self.class.user}"
+            GitReflow.say "\tEndpoint: #{self.class.api_endpoint}"
```

Comment:
One thing I noticed was that we are consistently using `GitReflow.say` because `git_reflow` includes `sandbox`. This can get a bit ugly when it says `GitReflow.say` everywhere, so another solution would be to directly include `sandbox` in the class or module where you are using `GitReflow.say`, but when I tried this, it required updating and remocking out many objects in Spec tests. This might not be worth it...